doctr/models/detection/linknet/pytorch.py

```diff
@@ -174,17 +178,26 @@ def compute_loss(
         seg_target, seg_mask = torch.from_numpy(seg_target).to(dtype=out_map.dtype), torch.from_numpy(seg_mask)
         seg_target, seg_mask = seg_target.to(out_map.device), seg_mask.to(out_map.device)
 
-        # BCE loss
+        # Focal loss
         bce_loss = F.binary_cross_entropy_with_logits(out_map, seg_target, reduction='none')
+        if gamma and gamma < 0:
```

Comment:
gamma is not None by default and a float, so I think we can safely keep `if gamma < 0:`